electron-app/src/components/molecules/ReportTool/FragmentLogs/FragmentLogs.jsx

```diff
@@ -1,65 +1,146 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
-import { Input } from 'antd';
+import { Input, Icon } from 'antd';
+import moment from 'moment';
 import ButtonPrimary from '../../../atoms/ButtonPrimary/ButtonPrimary';
 import { getMessage } from '../../../../utils/messages';
 import TableFragments from './TableFragments';
+import { getFragmentLogs } from '../../../../utils/api';
+import { formatDateTimeWithComma } from '../../../../utils/formatters';
 
-const FragmentLogs = ({
-  fragments,
-  currentPage,
-  totalPages,
-  onSearch,
-  onBackPage,
-  onNextPage
-}) => (
-  <div className="card">
-    <div className="titleCard4">
-      <h4>{getMessage('report.fragments.title')}</h4>
-      <Input placeholder={getMessage('report.fragments.search.placeholder')} />
-      <ButtonPrimary
-        text={getMessage('report.fragments.search.update')}
-        onClick={onSearch}
-      />
-    </div>
-    <TableFragments fragments={fragments} />
-    <div className="bottomBar">
-      <ButtonPrimary
-        text={getMessage('report.fragments.pagination.back')}
-        onClick={onBackPage}
-      />
-      <p>
-        {getMessage('report.fragments.pagination.current')}
-        {currentPage}
-        {getMessage('report.fragments.pagination.total')}
-        {totalPages}
-      </p>
-      <ButtonPrimary
-        text={getMessage('report.fragments.pagination.next')}
-        onClick={onNextPage}
-      />
-    </div>
-  </div>
-);
+const FragmentLogs = ({ nodeAddress }) => {
+  const PAGE_SIZE = 8;
+
+  const [fragmentLogs, setFragmentLogs] = useState();
+  const [inputFragmentId, setInputFragmentId] = useState();
+  const [errorMessage, setErrorMessage] = useState(
+    getMessage('report.fragments.missing')
+  );
+  const [currentPage, setCurrentPage] = useState(1);
+  const [totalPages, setTotalPages] = useState(1);
+
+  const fetchData = async () => {
```

Comment:
No necessary to this PoC, but when component reach some complexity, is a good practice to divide view from logic. ;)